janrain_capture.install

```diff
@@ -103,6 +103,40 @@ function janrain_capture_install() {
 
   // Set the default version
   variable_set('janrain_capture_ver', JANRAIN_CAPTURE_VERSION_DEFAULT);
+
```

Comment:
Please move this new code into update hook, e.g. `janrain_capture_update_7001($sandbox);`. Majority of markets Janrain Capture module is already enabled. It makes no sense to have this in hook_install for Janrain Capture module, because during this module installation, there is no Janrain users yet.